src/petals/cli/run_server.py

```diff
@@ -131,6 +131,7 @@ def main():
     parser.add_argument('--load_in_8bit', type=str, default=None,
                         help="Convert the loaded model into mixed-8bit quantized model. "
                              "Default: True if GPU is available. Use `--load_in_8bit False` to disable this")
+    parser.add_argument("--tensor_parallel_devices", nargs='+', default=[])  # disabled by default
```

Comment:
Seems like this feature is enabled by default, so please remove the comment. Also, please add the help string.